pycbc/workflow/pegasus_workflow.py

```diff
@@ -583,9 +583,7 @@ def save(self, filename=None, submit_now=False, plan_now=False,
                     # There was no storage path
                     pass
 
-        if not (submit_now or plan_now):
-            self._adag.write(filename)
-
```

Comment:
This is a comment for line 610, which I can't directly comment on: When we add this, we should also add it to "additional_planner_args.dat" so that when you set these things in the config file, they are also set when using `pycbc_submit_dax`. Then remove these options from `submit_dax` (working towards `submit_dax` basically doing very little).

Reply:
Ok, makes sense. I've added it to planner args and remove the dax option from pycbc_submit_dax